src/assertions/focus.js

```diff
@@ -0,0 +1,21 @@
+import elementExists from '../util/element-exists';
+
+export default function focus(client, chai, utils) {
+    chai.Assertion.addMethod('focus', function() {
+        const selector =  utils.flag(this, 'object');
+        const negate = utils.flag(this, 'negate');
+        const immediately = utils.flag(this, 'immediately');
+
+        if (!immediately) {
+            elementExists(client, selector, negate);
```

Comment:
In the case of focus the assertion should pass if it's negated.

Considering if I do:
```
expect('#test').to.not.have.focus();
```

And there is no element with #test on the page, it should pass because what is not there doesn't have focus either. But it should probably wait for 5s to make sure it doesn't show up later because of some async action finishing. 

What do you think?